### ISSUE TITLE
Get-vRABusinessGroupName filter by Name fails on special characters

### ISSUE THREAD

**Issue by sammcgeown** [jakkulabs/PowervRA#121]
Steps to reproduce
------------------
Connect-vRAServer -Server ServerName -Tenant TenantName -User Username
Get-vRABusinessGroupName -Name "Test & in Name"

Expected behavior
-----------------
Returns the business group or warns that it can't find it.

Actual behavior
---------------
Invoke-RestMethod fails with "System request parameter $filter=name eq 'Name is not properly formatted."

Environment data
----------------

```powershell
> $PSVersionTable

> $vRAConnection

```

**Comment by chelnak** [comment]
Could you test with %26 in place of the &?

**Comment by sammcgeown** [comment]
Yes - that works and returns the group as expected

**Comment by chelnak** [comment]
Excellent. Thanks.

**Comment by chelnak** [comment]
Will be included in v2.1.0! Thanks @sammcgeown 

```PowerShell
C:\PowervRA [development ≡ +0 ~2 -0 !]> Get-vRABusinessGroup -Name "Hello & Hello"

Name             : Hello & Hello
ID               : 84b139f1-f603-492e-9941-497929ca41dc
Description      : encoding test
Roles            :
ExtensionData    : @{entries=System.Object[]}
GroupManagerRole : {@{domain=vsphere.local; name=cgumbley}}
SupportUserRole  : {}
UserRole         : {}
Tenant           : Tenant01
```
